apps/baseline/models.py

```diff
@@ -227,7 +227,15 @@ class Language(models.TextChoices):
         verbose_name=_("Population Estimate"),
         help_text=_("The estimated population of the Livelihood Zone during the reference year"),
     )
-
+    currency = models.ForeignKey(
+        Currency,
+        blank=True,
```

Comment:
Why `blank=True`. I don't think we can have a Baseline without a default currency, because the LivelihoodActivity records need a currency, and it normally isn't specified there.

I think we could just make it mandatory here, and then set the migration to default them all to USD, say, knowing that we will have to fix them anyway.